plugins/cue_points/quiz/lib/kQuizManager.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @package plugins.quiz
+ * @subpackage quiz
+ */
+class kQuizManager implements kObjectChangedEventConsumer
+{
+	const EVENT_TYPE = 'eventType';
+	const QUIZ_EVENT_TYPE = '@event_type_id@';
+	const KUSER_ID = 'kuserId';
+	const ENTRY_ID = 'entryId';
+	const PARTNER_ID = 'partnerId';
+	const VERSION = 'version';
+	const ANSWER_IDS = 'answerIds';
+	const SCORE = 'score';
+	const NUM_OF_CORRECT_ANSWERS = 'numOfCorrectAnswers';
+	const NUM_OF_QUESTIONS = 'numOfQuestions';
+	const NUM_OF_RELEVANT_QUESTIONS = 'numOfRelevnatQuestions';
+	const CALCULATED_SCORE = 'calculatedScore';
+	const INTERNAL_ANALYTICS_HOST = 'internal_analytics_host';
+	
+	/**
+	 * @param BaseObject $object
+	 * @param array $modifiedColumns
+	 * @return bool
+	 */
+	public function objectChanged (BaseObject $object, array $modifiedColumns)
+	{
+		if (!self::wasStatusChanged($object, $modifiedColumns))
+		{
+			return false;
+		}
+		$quizEventContent = self::getQuizEventContent($object);
```

Comment:
move this under the if, there is no point getting the content if we are not going to send the beacon